src/converse-core.js

```diff
@@ -335,6 +335,7 @@
             sid: undefined,
             sounds_path: '/sounds/',
             storage: 'session',
+            synchronize_presence: true, // Set to false to not sync with other clients or with resource name of the particular client that it should synchronize with
             use_vcards: true,
```

Comment:
Actually I have not tried with multiple instances of converse.js in the same browser.

give a couple of  minutes and i'll update pull request with your inline suggestions.

Thanks!